tools/build_config/build.sh

```diff
@@ -35,14 +35,16 @@ function run_configure() {
   local build_dir=$1
   local build_config=$2
   local build_tests=$3
-  local cmake_flags="${4-}"  # Default value is empty string
+  local matlab_release=$4
+  local cmake_flags="${5-}"  # Default value is empty string
 
   cmake_cmd="cmake ${HERBERT_ROOT}"
   cmake_cmd+=" -G \"${CMAKE_GENERATOR}\""
   cmake_cmd+=" -DMatlab_ROOT_DIR=${MATLAB_ROOT}"
   cmake_cmd+=" -DCMAKE_BUILD_TYPE=${build_config}"
   cmake_cmd+=" -DBUILD_TESTS=${build_tests}"
   cmake_cmd+=" -DBUILD_FORTRAN=${build_fortran}"
+  cmake_cmd+=" -DMatlab_RELEASE=${matlab_release}"
```

Comment:
have you tested that this fails if `Matlab_RELEASE` is inconsistent with `MATLAB_ROOT`?

Reply:
It throws an error the first time - not the second. Fix upcoming